.github/workflows/nightly.yml

```diff
@@ -0,0 +1,29 @@
+name: Nightly Build of Reports Website
```

Comment:
Just thinking, but if we're adding a Rust project we should also add Rust checks to the CI. So something that builds `generator` on PRs and checks for clippy warnings.

i.e: https://github.com/Rust-GCC/bottleboard/blob/main/.github/workflows/check.yml or https://github.com/Rust-GCC/testing/blob/main/.github/workflows/build_check.yml